virt/install/installing-ove.adoc

```diff
@@ -0,0 +1,48 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="installing-ove"]
+= Installing an {ove} cluster
+:context: installing-ove
+
+toc::[]
+
+The following procedures describe a streamlined method to install a {ove-first} cluster that requires less planning and configuration compared to other installation methods.
```

Comment:
I gently disagree on the motivations provided :) 

The feature keypoints here are:
* This installation method doesn't require the user to setup an external registry in  a disconnected or air-gapped environment (as required instead by all the other disconnected installation approachs)
* The user-friendly UI makes the installation simpler (but the amount of the configuration provided at the end doesn't change), since it provides a useful number of hints/warnings and guides the process via a clear and well defined wizard

These points are also mentioned below, but IMHO they deserve more focus and relevance (especially the registry-less one)

Reply:
Great, thanks for this insight! I didn't really know how to present this method (especially since it doesn't have some official name), so any help about how to frame this feature is appreciated.